src/cpp/fastdds/xtypes/type_representation/TypeObjectRegistry.cpp

```diff
@@ -438,15 +438,6 @@ ReturnCode_t TypeObjectRegistry::register_type_object(
         const TypeObject& type_object)
 {
     uint32_t type_object_serialized_size = 0;
-    try
-    {
-        TypeObjectUtils::type_object_consistency(type_object);
-    }
-    catch (const InvalidArgumentError& exception)
-    {
-        EPROSIMA_LOG_ERROR(XTYPES_TYPE_REPRESENTATION, "Inconsistent CompleteTypeObject: " << exception.what());
-        return eprosima::fastdds::dds::RETCODE_PRECONDITION_NOT_MET;
-    }
```

Comment:
Please update the method documentation. I think it should be documented that TypeObject consistency is not checked due to the dependencies being probably received unordered and that the registering consistency is responsibility of the type lookup service.

Reply:
updated in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)